core/src/com/po/fuck/view/HealthBar.java

```diff
@@ -0,0 +1,35 @@
+package com.po.fuck.view;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.Entity;
+
+import static com.po.fuck.view.Constants.HEALTHBAR_HEIGHT;
+import static com.po.fuck.view.Constants.HEALTHBAR_WIDTH;
+import static com.po.fuck.view.Constants.HEALTHBAR_OFFSET;
+import static com.po.fuck.view.DrawableMisc.createTexture;
+
+public final class HealthBar {
+    protected Sprite healthBarSprite;
+    protected Sprite backgroundSprite;
+
+    private Entity entity;
+
+    public HealthBar(Entity entity) {
+        this.entity = entity;
+    }
+
+    public <Drawer extends CenterDrawer> void draw(Drawer drawer, float spriteHeight){
```

Comment:
Maybe rename `spriteHeight` into `entityHeight`, because it sounds like you are talking about `HealthBar`'s sprite ?